packages/experimental/Link/src/useLink.ts

```diff
@@ -60,14 +64,27 @@ export const useLink = (props: LinkProps): LinkInfo => {
 
   const linkTooltip = tooltip ?? url ?? undefined;
 
+  /*These callbacks are not implemented on iOS/macOS, and cause Redboxes if passed in. Limit to only windows/win32 for now*/
+  const filteredProps = {
```

Comment:
Are these filteredProps the same as in Text.tsx? Should it be moved to some helper file to avoid duplication?

Reply:
Currently, filteredProps in Text doesn't include the MouseOver events. I'm not sure how long these would be filtered out for, but if Text.tsx ends up supporting mouseOver events, I'll move them out to a shared location!

Reply:
I don't think we on the macOS side have any plans to implement the missing callbacks for a while. And even if we do, Android and iOS wouldn't get them. It makes sense to keep these filtered props to just the Text component to me though, since it's Text specific. 

Previously, adapters did this for us but 🤷🏾